subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/impl/DirectorySnapshotterTest.groovy

```diff
@@ -85,6 +87,20 @@ class DirectorySnapshotterTest extends Specification {
         ].collect { 'root/' + it }) as Set
     }
 
+    def "should snapshot file system root"() {
+        given:
+        def fileSystemRoot = fileSystemRoot()
+        def patterns = new PatternSet().exclude("*")
```

Comment:
@wolfs, `"*"` suffices, right? Or do I need `"**"`?

Reply:
Good question. If `*` works, go for it! From my understand it should...